docker/traefik/docker-compose.yml

```diff
@@ -25,27 +25,27 @@ services:
     depends_on:
       - db
     environment:
-      VOCASCAN_CONFIG: "/etc/vocascan/vocascan.config.js"
+      VOCASCAN_CONFIG: '/etc/vocascan/vocascan.config.js'
     volumes:
-      - "./vocascan.config.js:/etc/vocascan/vocascan.config.js:ro"
+      - './vocascan.config.js:/etc/vocascan/vocascan.config.js:ro'
     labels:
-      - "traefik.http.routers.vocascan.rule=Host(`web.your_domain.com`)"
-      - "traefik.http.routers.vocascan.tls=true"
-      - "traefik.http.routers.vocascan.tls.certresolver=lets-encrypt"
-      - "traefik.http.services.vocascan.loadbalancer.server.port=8000"
+      - 'traefik.http.routers.vocascan.rule=Host(`your_domain.vocascan.com`) && PathPrefix(`/p/`, `/api`)''
```

Comment:
You accidentally changed to vocascan. And secondly, are those two quotes at the end of the line correct?